lib/v2/weibo/search/hot.js

```diff
@@ -15,21 +18,108 @@ module.exports = async (ctx) => {
+        });
+
+        const resultItems = await Promise.all(
+            tlist.map((i) =>
+                ctx.cache.tryGet(i.plink, async () => {
+                    const pInfo = await fetchContent(i.plink);
+                    i.description = pInfo.content;
+                    return i;
+                })
+            )
+        );
+        ctx.state.data = {
+            title: '微博热搜榜',
+            link: 'https://s.weibo.com/top/summary?cate=realtimehot',
+            description: '实时热点，每分钟更新一次',
+            item: resultItems,
+        };
+    } else {
+        ctx.state.data = {
+            title: '微博热搜榜',
+            link: 'https://s.weibo.com/top/summary?cate=realtimehot',
+            description: '实时热点，每分钟更新一次',
+            item: data.cards[0].card_group.map((item) => {
+                const title = item.desc;
+                const link = `https://m.weibo.cn/search?containerid=100103type%3D1%26q%3D${encodeURIComponent(item.desc)}`;
+                const description = item.desc;
+
+                return {
+                    title,
+                    description,
+                    link,
+                };
+            }),
+        };
+    }
 };
+
+async function fetchContent(url) {
+    // Fetch the subpageinof
+    const cookieString = config.weibo.cookies ? config.weibo.cookies : '';
+    const subres = await got(url, {
+        headers: {
+            Cookie: cookieString,
+        },
+    });
+    let demostr = '';
+    try {
+        const rdata = JSON.parse(subres.body);
+        const cards = rdata.data.cards;
+        // Need to find one cards with 'type ==9'
+        demostr = seekContent(cards);
+    } catch (e) {
+        // console.log(e);
+        // console.log(url);
+    }
+    const ret = demostr;
+    return {
+        content: ret,
+    };
+}
+
+function seekContent(clist) {
+    const $ = cheerio.load('<div id="wbcontent"></div>');
+    const stub = $('#wbcontent');
+
+    // Need to find one clist with 'type ==9'
+    for (let i = 0; i < clist.length; i++) {
+        const curitem = clist[i];
+        if (curitem.card_type === 9) {
+            const curcontent = cheerio.load(curitem.mblog.text);
+            //            curcontent('img').attr('width', '1em').attr('height', '1em');
+            curcontent('img').remove();
+            const section = art(path.join(__dirname, 'template/digest.art'), {
+                author: {
+                    link: curitem.mblog.user.profile_url,
+                    name: curitem.mblog.user.screen_name,
+                },
+                msg: curcontent.html(),
+                link: curitem.scheme,
+                postinfo: curitem.mblog.created_at,
+                //               picnum: curitem.mblog.pic_num,
+                //               pics:   curitem.mblog.pics,
+            });
+            stub.append(section);
+        }
+        if (curitem.card_type === 11) {
+            stub.append(seekContent(curitem.card_group));
+        }
+    }
```

Comment:
You can use [for...of](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/for...of)